### ISSUE TITLE
man page: move "backintime-config (1)" into "backintime-config (5)"

### ISSUE THREAD

**Issue by buhtz** [bit-team/backintime#1773]
Blocked by
- #1923
- #2085

The "backintime-config" man page describe the config file format for Back In Time. That man page lives in section 1 "Executable programs or shell commands" but should live in section 5 "File formats and conventions, e.g. /etc/passwd".
Also adapt the man page links in the other man pages.

See [`man man`](https://man7.org/linux/man-pages/man1/man.1.html) for details about man page sections.

I was searching the issues. Haven't we discussed this in the past?

I would say we don't need it but if someone is concerned about backwards compatibility we could add a backintime-config.1 man page just mention the movement to backintime-config.5.

See also
- #1772

**Comment by aryoda** [comment]
> I was searching the issues. Haven't we discussed this in the past?

Surprisingly not :-)

> I would say we don't need it but if someone is concerned about backwards compatibility
> we could add a backintime-config.1 man page just mention the movement to backintime-config.5

I am not concerned and think unless someone explicitly uses the man page section number to search for man pages (and therefore should be an advanced user) it still works as before.

**Comment by emtiu** [comment]
> I am not concerned and think unless someone explicitly uses the man page section number to search for man pages (and therefore should be an advanced user) it still works as before.

I agree, I don't think we need to maintain an empty `backintime-config.1` man page.
